src/interfaces/libpq/ivytest/oracle_test.sh

```
@@ -0,0 +1,168 @@
+		LISTEN_ADDRESSES=""
+		# Select a socket directory.  The algorithm is from the "configure"
+		# script; the outcome mimics pg_regress.c:make_temp_sockdir().
+		PGHOST=$PG_REGRESS_SOCK_DIR
+		if [ "x$PGHOST" = x ]; then
+			{
+				dir=`(umask 077 &&
+					  mktemp -d /tmp/ivytest_check-XXXXXX) 2>/dev/null` &&
+				[ -d "$dir" ]
+			} ||
+			{
+				dir=/tmp/ivytest_check-$$-$RANDOM
```

Shellcheck (0.10.0)

[warning] In POSIX sh, RANDOM is undefined.

(SC3028)